packages/ui5-proxy-middleware/src/base/ui5.d.ts

```diff
@@ -0,0 +1,118 @@
+declare module '@ui5/fs' {
+    /**
+     * https://sap.github.io/ui5-tooling/api/module-@ui5_fs.Resource.html
+     */
+    export class Resource {
+        /**
+         * Gets the resources path
+         */
+        getPath(): string;
+
+        /**
+         * Gets a buffer with the resource content.
+         */
+        getBuffer(): Promise<Buffer>;
+
+        /**
+         * Gets a string with the resource content.
+         */
+        getString(): Promise<string>;
+    }
+
+    /**
+     * https://sap.github.io/ui5-tooling/stable/api/@ui5_fs_ReaderCollection.html
```

Comment:
```suggestion
     * https://sap.github.io/ui5-tooling/v3/api/@ui5_fs_ReaderCollection.html
```
Broken link. This is the v3 link